### ISSUE TITLE
New node unable to sync

### ISSUE THREAD

**Issue by manihagh** [energywebfoundation/ewc-system-tests#18]
Starting parity clean with no db and get this after a while:

```Starting Parity-Ethereum/v2.3.9-stable-0b428262e-20190401/x86_64-macos/rustc1.33.0
2019-04-17 16:03:53  Keys path /Users/mani/Library/Application Support/io.parity.ethereum/keys/Volta
2019-04-17 16:03:53  DB path /Users/mani/Library/Application Support/io.parity.ethereum/chains/Volta/db/c4f2321e28d1c489
2019-04-17 16:03:53  State DB configuration: fast
2019-04-17 16:03:53  Operating mode: active
2019-04-17 16:03:53  Configured for Volta using AuthorityRound engine
2019-04-17 16:03:53  Public node URL: enode://5579cf23592bba64a81591b3d4a8590e3a4d24b802674be5d52694a60d968a458c84b73a2cdf8d3c809933a6edcf7267b4ebf3cf25d671172fa89a49e0b85375@192.168.178.40:30303
2019-04-17 16:04:03  Syncing    #4224 0xe079…9887   418.57 blk/s   25.4 tx/s    0.7 Mgas/s      0+ 5425 Qed     #9779    1/25 peers   1 MiB chain 88 MiB db 8 MiB queue 1 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-17 16:04:08  Syncing    #9278 0x23ef…f45a  1009.79 blk/s    0.0 tx/s    0.0 Mgas/s      0+14089 Qed    #23368    1/25 peers   2 MiB chain 89 MiB db 20 MiB queue 3 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-17 16:04:12  Signal for transition within contract. New list: [0x7e8b8661dbc77d6bee7a1892fbcf8ef6378cab30, 0xdae561c716f9ea58e32e37d9ae95465eca286012, 0xebee2fc556975c3dd50c17d13a15af535fb7bbb3, 0x1e4a5b32eef4938a8b46fc4a7824f44b3e5de3ec]
2019-04-17 16:04:12  Applying validator set change signalled at block 13595
2019-04-17 16:04:13  Syncing   #14341 0x3edb…910d  1011.99 blk/s    9.8 tx/s    0.8 Mgas/s      0+16063 Qed    #30406    1/25 peers   3 MiB chain 89 MiB db 23 MiB queue 4 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-17 16:04:18  Syncing   #19088 0x894e…c778   948.83 blk/s    2.2 tx/s    0.7 Mgas/s      0+11312 Qed    #30407    1/25 peers   5 MiB chain 88 MiB db 16 MiB queue 4 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-17 16:04:23  Syncing   #23348 0x864e…b4b0   851.66 blk/s    0.0 tx/s    0.0 Mgas/s      0+ 7053 Qed    #30408    1/25 peers   5 MiB chain 88 MiB db 10 MiB queue 4 MiB sync  RPC:  0 conn,  236 req/s,  143 µs
2019-04-17 16:04:28  Syncing   #26543 0x2fb8…67cd   638.11 blk/s    0.0 tx/s    0.0 Mgas/s      0+ 3862 Qed    #30409    1/25 peers   7 MiB chain 88 MiB db 6 MiB queue 4 MiB sync  RPC:  0 conn,  377 req/s,  155 µs
2019-04-17 16:04:33  Syncing   #29064 0x4c14…cfe2   504.10 blk/s    2.2 tx/s    0.6 Mgas/s      0+ 1343 Qed    #30410    1/25 peers   6 MiB chain 87 MiB db 2 MiB queue 4 MiB sync  RPC:  0 conn,  262 req/s,  142 µs
2019-04-17 16:04:36  Imported #30411 0xc9b8…3318 (0 txs, 0.00 Mgas, 33 ms, 0.55 KiB) + another 21 block(s) containing 8 tx(s)
2019-04-17 16:04:40  Imported #30412 0x0123…843d (0 txs, 0.00 Mgas, 2 ms, 0.55 KiB)
2019-04-17 16:04:44  Stage 1 block verification failed for 0x97c0…e464: Error(Block(TimestampOverflow), State { next_error: None, backtrace: InternalBacktrace { backtrace: None } })
2019-04-17 16:04:44  
Bad block detected: TimestampOverflow
RLP: f9022cf90227a0012398abb36a4e1e36582270b68c24a07b1e3404ee297ef050ef1bbe695d843da01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d4934794dae561c716f9ea58e32e37d9ae95465eca286012a02749754316f68e267cb14da2b8627f57c5a0b5ad27a030b830c872babf1a1b9da056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421b901000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000090fffffffffffffffffffffffffffffffe8276cd837a120080845cb7327d8367737984128b0a19b8416d8607d3c3572e7c780db97546e48209ad2cd399a1a037713a4695895025eff563bcd1de516e03fea989c801c8a638379738795e33e98f38f4dcde0ce254610900c0c0
Header: Header { parent_hash: 0x012398abb36a4e1e36582270b68c24a07b1e3404ee297ef050ef1bbe695d843d, timestamp: 1555509885, number: 30413, author: 0xdae561c716f9ea58e32e37d9ae95465eca286012, transactions_root: 0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421, uncles_hash: 0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347, extra_data: [103, 115, 121], state_root: 0x2749754316f68e267cb14da2b8627f57c5a0b5ad27a030b830c872babf1a1b9d, receipts_root: 0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421, log_bloom: 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, gas_used: 0, gas_limit: 8000000, difficulty: 340282366920938463463374607431768211454, seal: [[132, 18, 139, 10, 25], [184, 65, 109, 134, 7, 211, 195, 87, 46, 124, 120, 13, 185, 117, 70, 228, 130, 9, 173, 44, 211, 153, 161, 160, 55, 113, 58, 70, 149, 137, 80, 37, 239, 245, 99, 188, 209, 222, 81, 110, 3, 254, 169, 137, 200, 1, 200, 166, 56, 55, 151, 56, 121, 94, 51, 233, 143, 56, 244, 220, 222, 12, 226, 84, 97, 9, 0]], hash: Some(0x97c02d7975a5addcac524c2a360bc0a30fac31f91be9f292b2286e0c8272e464) }
Uncles: 
Transactions:

2019-04-17 16:05:03     0/25 peers   0 bytes chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,    3 req/s,   51 µs
2019-04-17 16:05:33     0/25 peers   42 KiB chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,   14 req/s,   30 µs
2019-04-17 16:06:03     0/25 peers   42 KiB chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,    1 req/s,   25 µs
2019-04-17 16:06:33     0/25 peers   322 KiB chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,    1 req/s,   24 µs
2019-04-17 16:07:03     0/25 peers   386 KiB chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,    1 req/s,   31 µs
2019-04-17 16:07:33     0/25 peers   386 KiB chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,    1 req/s,   20 µs
2019-04-17 16:08:03     0/25 peers   386 KiB chain 87 MiB db 0 bytes queue 4 MiB sync  RPC:  0 conn,    1 req/s,   21 µs```

**Comment by matthiasexperimental** [comment]
I had the same problems with a global poa network with nodes in aws and azure. The network uses dynamic validators over a validator contract. Every node on the cloud worked without problems. My local parity node on mac runs very instable. After I install chronyd on my mac and synchronize the time not over the apple time server, everything works fine.

The error messages did depend on the parity version I used. If I use the last stable, I did get “Bad block detected: TimestampOverflow with a blocktrace”. If I used v2.3.3 I did get “Bad block detected: TemporarilyInvalid(OutOfBounds”.

With v2.4.5 I lost also every time that happened the peers.

I hope this info’s are use full.
